plugin.py

```diff
@@ -608,6 +618,42 @@ def onHeartbeat(self):
         else:
             self.mqttClient.Ping()
 
+        # Sensor support
+        # Timing out sensors
+        #Domoticz.Debug( "OnHeartbeat: Settings " + str( Settings ) )
+
+        now            = datetime.now()
+        update_timeout = int(Settings['SensorTimeout'])
+
+        #Domoticz.Debug( "OnHeartbeat: " + str( Devices.items() ) + " " + str( type( Devices ) ) )
+
+        for k, device in Devices.items():
```

Comment:
`k` is not used? Loop over `Devices.values()` instead

Reply:
This is a copy/paste from your original code. Prefer to leave intact :)